Readability.js

```diff
@@ -721,11 +721,11 @@ Readability.prototype = {
           }
         }
 
-        // Remove empty DIV, SECTION, and HEADER nodes
+        // Remove DIV, SECTION, and HEADER nodes without any content(e.g. text, image, video, or iframe).
```

Comment:
Sure, let's do it. If we find out more superfluous we want to remove in the future, we can list them at that time.

Already updated the patch for it, please take a look, thanks. :)